examples/configure_hydra/workdir/conf/config.yaml

```diff
@@ -5,3 +5,5 @@ hydra:
   sweep:
     dir: sweep_dir
     subdir: ${hydra.job.num}
+
+a: a1
```

Comment:
Why change this?

Reply:
in the corresponding example doc, I want to do `python my_app.py a=a1,a2,a3` without the `+` to make it cleaner.

Reply:
Gotcha.